rtengine/color.cc

```diff
@@ -2117,22 +2058,23 @@ void Color::skinred ( double J, double h, double sres, double Sp, float dred, fl
             }
         }
 
-        if(chromapro > 0.0) {
+        constexpr float deltaHH = 0.3f; //HH value transition : I have choice 0.3 radians
+        if(chromapro > 0.f) {
             Color::scalered ( rstprotection, chromapro, 0.0, HH, deltaHH, scale, scaleext);    //Scale factor
         }
 
-        if(chromapro > 1.0) {
-            interm = (chromapro - 1.0f) * 100.0f;
+        float factorskinext;
+        if(chromapro > 1.f) {
+            const float interm = (chromapro - 1.0f) * 100.0f;
             factorskin = 1.0f + (interm * scale) / 100.0f;
             factorskinext = 1.0f + (interm * scaleext) / 100.0f;
         } else {
             factorskin = chromapro ;
             factorskinext = chromapro ;
```

Comment:
This needs to be run through a linter...

Reply:
Function removed